src/status_im/ui/screens/discover/all_dapps/views.cljs

```diff
@@ -20,15 +20,12 @@
       [react/view styles/dapps-list-item-second-row
        [react/view styles/dapps-list-item-name-container
         [react/view styles/dapps-list-item-avatar-container
-         [react/view (chat-icon/contact-icon-contacts-tab dapp)]]
+         [react/view [chat-icon/contact-icon-view dapp {:size 80}]]]
```

Comment:
Is that `80` expected to be hardcoded?

Reply:
According to design it should be 2x, and 40 is hardcoded for rest of app

Reply:
Looks OK on iPhone 7Plus and not horrible on iPhone 5s. Might have to go to 2 rows or rethink icon size for small resolutions, will check with QA/design next call.